cmd/cloud-init_test.go

```diff
@@ -49,7 +49,7 @@ var _ = Describe("cloud-init", Label("cloud-init", "cmd"), func() {
 					"-s",
 					"tests",
 					"-d",
-					`stages.tests=foo`,
+					`'stages.tests=foo'`,
```

Comment:
nit: should we keep the same format for quoting ?
```suggestion
					"'stages.tests=foo'",
```